assets/javascripts/discourse-markdown/bbcode.js.es6

```diff
@@ -357,9 +357,40 @@ function setupMarkdownIt(md) {
     }
   });
 
+  md.block.bbcode.ruler.push("font", {
```

Comment:
Here's the real code additions. Running formatting in VSCode adds all le spaces.